services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentContentServiceITest.kt

```diff
@@ -83,15 +83,17 @@ class ApplicationAttachmentContentServiceITest(
         fun `Doesn't throw an error if content is missing`() {
             val attachment = attachmentWithoutCloudContent().copy(blobLocation = "missing")
 
-            assertThat(runCatching { attachmentContentService.delete(attachment) }).isSuccess()
+            assertThat(runCatching { attachmentContentService.delete(attachment.blobLocation!!) })
+                .isSuccess()
```

Comment:
This seems overly complicated, the test will fail anyway if there's an unexpected exception.
```suggestion
            attachmentContentService.delete(attachment.blobLocation!!)
```

Reply:
Adjusted the test to test whether anything is actually deleted or not.